Lib/logging/handlers.py

```diff
@@ -815,7 +815,10 @@ def __init__(self, address=('localhost', SYSLOG_UDP_PORT),
 
         if isinstance(address, str):
             self.unixsocket = True
-            self._connect_unixsocket(address)
+            try:
+                self._connect_unixsocket(address)
+            except OSError:
+                pass
```

Comment:
This is pretty surprising to read—it makes more sense with the comment from the [associated bug report][bug].

Would you be averse to adding a comment in here to explain why it's safe to suppress this error?

[bug]: http://bugs.python.org/issue29808